### PR TITLE
Fix unexpected max latency in output of `stat`

### LINKED ISSUE #348: Fix unexpected max latency in output of `stat`
### RaftKeeper version
2.1.1
### Description
Got a large number of max latency
```
echo stat|nc localhost 2181
RaftKeeper version: RaftKeeper v2.1.1
Clients:
 127.0.0.1:11518(recved=0,sent=0)
 11.118.148.89:61362(recved=323087487,sent=325752485)
 11.118.149.166:26454(recved=835632782,sent=842340316)
 11.118.249.188:12130(recved=899254869,sent=906579323)
 11.115.152.197:22470(recved=890502441,sent=897937851)
 11.118.71.54:46136(recved=924492875,sent=931955872)
 11.118.135.8:53452(recved=943845776,sent=951538936)
 11.102.65.60:18428(recved=949945813,sent=957763382)
 11.102.64.186:31594(recved=1020374675,sent=1028727434)
 11.118.125.232:29714(recved=866264742,sent=873505846)
 11.102.65.67:22876(recved=1122690863,sent=1132029069)
 11.102.64.101:20116(recved=1250031662,sent=1260370136)
 11.134.48.159:27372(recved=892861462,sent=900254139)

Latency min/avg/max: 0/0/18446744073709551615
```
18446744073709551615 which is 0xffffffffffffff meaning -1 for int64

### PR DESCRIPTION
**Pull request by lzydmxy**
### Which issues of this PR fixes:

This PR try to fix #348

### CODE REVIEW

File: src/Service/ConnectionHandler.cpp [hunk 1]
```diff
@@ -687,20 +687,34 @@ void ConnectionHandler::updateStats(const Coordination::ZooKeeperResponsePtr & r
     if (response->xid != Coordination::WATCH_XID && response->getOpNum() != Coordination::OpNum::Heartbeat
         && response->getOpNum() != Coordination::OpNum::SetWatches && response->getOpNum() != Coordination::OpNum::Close)
     {
-        Int64 elapsed = Poco::Timestamp().epochMicroseconds() / 1000 - response->request_created_time_ms;
+        auto current_time = getCurrentTimeMilliseconds();
+        Int64 elapsed = current_time - response->request_created_time_ms;
+        if (elapsed < 0)
```
Comment (JackyWoo) [inline src/Service/ConnectionHandler.cpp:692]: Why elapsed can be -1? Caused by different clock? If it is, we must use `stedy_lock` in all places.

Comment (lzydmxy) [reply src/Service/ConnectionHandler.cpp:692]: Maybe different clock, so I use `getCurrentTimeMilliseconds` which is `steady_clock` for request in all places.

Comment (JackyWoo) [reply src/Service/ConnectionHandler.cpp:692]: Maybe caused by clock diff in different server. When leader append_entries the `create_time` is attached to the log and when the follower take the log the time in server is used. If there is a clock diff in the servers `-1` may raise.

Comment (JackyWoo) [reply src/Service/ConnectionHandler.cpp:692]: Let's add a coment to help reader understand the code.

File: src/Service/ConnectionHandler.cpp [hunk 2]
```diff
@@ -687,20 +687,34 @@ void ConnectionHandler::updateStats(const Coordination::ZooKeeperResponsePtr & r
...
-            conn_stats.updateLatency(elapsed);
-            if (unlikely(elapsed > 10000))
-                LOG_WARNING(
-                    log,
-                    "Slow request detected #{}#{}#{}, time costs {}ms, please take care.",
-                    toHexString(session_id.load()),
-                    response->xid,
-                    Coordination::toString(response->getOpNum()),
-                    elapsed);
+            LOG_WARNING(
```
Comment (JackyWoo) [inline src/Service/ConnectionHandler.cpp:694]: Just delete the log, it is confused when system mantainers see the log.

Comment (lzydmxy) [reply src/Service/ConnectionHandler.cpp:694]: okay

File: src/Service/ConnectionHandler.cpp (hunk 1 above)
Comment (lzydmxy) [reply src/Service/ConnectionHandler.cpp:692]: > Maybe caused by clock diff in different servers. When leader append_entries the `create_time` is attached to the log and when the follower takes the log the timestamp in leader is used. If there is a clock diff in the servers `-1` may raise.

When followers takes the log from leader,  them won't updateLatency for the log.We used to use `system_clock` to measure request time cost.

**Pull request merged**